enterprise/api/v1/views.py

```diff
@@ -76,39 +108,34 @@ def courses(self, request, pk=None):
 
         Only courses with active course runs are returned. A course run is considered active if it is currently
         open for enrollment, or will open in the future.
-        ---
-        serializer: serializers.CourseSerializerExcludingClosedRuns
         """
+
         enterprise_customer = self.get_object()
         self.check_object_permissions(request, enterprise_customer)
-
-        # Make sure there is a catalog associated with the enterprise
-        if not enterprise_customer.catalog:
-            error_message = (
-                "No catalog is associated with Enterprise {enterprise_name} from endpoint '{path}'.".format(
-                    enterprise_name=enterprise_customer.name,
-                    path=request.get_full_path()
-                )
+        self.ensure_data_exists(
+            request,
+            enterprise_customer.catalog,
+            "No catalog is associated with Enterprise {enterprise_name} from endpoint '{path}'.".format(
```

Comment:
@UmanShahzad Can we pass this string as a kwarg to make it even easier to tell what its purpose is?